aws/resource_aws_wafv2_ip_set_test.go

```diff
@@ -374,7 +375,7 @@ resource "aws_wafv2_ip_set" "ip_set" {
   scope              = "REGIONAL"
   ip_address_version = "IPV6"
   addresses          = [
-    "0:0:0:0:0:ffff:7f00:1/64",
```

Comment:
If you enter `0:0:0:0:0:ffff:7f00:1/64` in the console it reports as `0000:0000:0000:0000:0000:0000:0000:0000/64` when you refresh. Replace with a more usual IPv6 CIDR.